orca-core/src/main/java/com/netflix/spinnaker/orca/ExecutionStatus.java

```diff
@@ -104,7 +104,7 @@ public final boolean isHalt() {
   public static final Collection<ExecutionStatus> COMPLETED = Collections.unmodifiableList(Arrays.asList(SUCCEEDED, STOPPED, SKIPPED, TERMINAL, FAILED_CONTINUE));
 
   private static final Collection<ExecutionStatus> SUCCESSFUL = Collections.unmodifiableList(Arrays.asList(SUCCEEDED, STOPPED, SKIPPED));
-  private static final Collection<ExecutionStatus> FAILURE = Collections.unmodifiableList(Arrays.asList(TERMINAL, STOPPED, FAILED_CONTINUE));
+  private static final Collection<ExecutionStatus> FAILURE = Collections.unmodifiableList(Arrays.asList(TERMINAL, STOPPED, FAILED_CONTINUE, CANCELED));
```

Comment:
`ExecutionStatis.isFailure()` is only called from `CompleteStageHandler` to gate calling `planOnFailureStages()`. Adding `CANCELED` here should allow failure stages to fully replace `CancellableStage`.